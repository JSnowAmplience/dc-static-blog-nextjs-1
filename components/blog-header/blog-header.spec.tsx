/* eslint-env jest */
import renderer from 'react-test-renderer';
import BlogHeader from './blog-header';

describe('BlogHeader', () => {
  test('renders blog header', async () => {
    const props = {
      title: 'Blog Title'
    };
    const component = await renderer.create(<BlogHeader {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
