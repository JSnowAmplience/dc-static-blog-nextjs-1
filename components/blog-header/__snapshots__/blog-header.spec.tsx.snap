// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogHeader renders blog header 1`] = `
<section
  className="jsx-775714430"
>
  <img
    alt="alt-tag"
    className="jsx-775714430 logo"
    src="/static/images/logo.png"
  />
  <h1
    className="jsx-775714430"
  >
    Blog Title
  </h1>
</section>
`;
